includes/admin/class-wc-payments-admin.php

```diff
@@ -65,7 +65,7 @@ public function __construct(
 
 		// Add menu items.
 		add_action( 'admin_menu', [ $this, 'add_payments_menu' ], 0 );
-		add_action( 'admin_menu', [ $this, 'maybe_redirect_to_onboarding' ], 1 );
+		add_action( 'admin_init', [ $this, 'maybe_redirect_to_onboarding' ], 11 ); // Run this after the WC setup wizard and onboarding redirection logic.
```

Comment:
With this change we might want to add the following code to the top of `maybe_redirect_to_onboarding` for added safety:

```php
if ( wp_doing_ajax() ) {
	return false;
}
```

Reply:
Sounds like a good idea.

Reply:
Included here: https://github.com/Automattic/woocommerce-payments/pull/2514/commits/1332331db725a5e931f51d5e8abc5664a73a1c75